src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -34,7 +34,7 @@ import {hideContextMenu, showDeleteModal} from './ReportActionContextMenu';
 
 /** Gets the HTML version of the message in an action */
 function getActionHtml(reportAction: OnyxEntry<ReportAction>): string {
-    const message = reportAction?.message?.at(-1) ?? null;
+    const message = Array.isArray(reportAction?.message) ? reportAction?.message?.at(-1) ?? null : reportAction?.message ?? null;
```

Comment:
Why not use `ReportActionsUtils.getOriginalMessage` here?

Reply:
We're getting the last element of `message` array field here so I don't think we can use `ReportActionsUtils.getOriginalMessage` in this case.